static/js/components/search/FilterVisibilityToggle.js

```diff
@@ -13,6 +13,7 @@ export const FILTER_ID_ADJUST = {
 
 export default class FilterVisibilityToggle extends SearchkitComponent {
   props: {
+    title:             string,
```

Comment:
can you fix the alignment for this prop? (with the indentation of the other props, just for readability)